Policies/APIManagement/Azure_APIManagement_AuthN_Disable_Management_API/Security - APIManagement - Audit API Management Service with Enabled Managament API.json

```diff
@@ -0,0 +1,38 @@
+{
+  "mode": "All",
+  "parameters": {
+    "effect": {
+      "type": "String",
+      "metadata": {
+        "displayName": "Effect",
+        "description": "The effect determines what happens when the policy rule is evaluated to match"
+      },
+      "allowedValues": [
+        "Disabled",
+        "AuditIfNotExists"
+      ],
+      "defaultValue": "AuditIfNotExists"
+    }
+  },
+  "policyRule": {
+    "if": {
+      "field": "type",
+      "equals": "Microsoft.ApiManagement/service"
+    },
+    "then": {
+      "effect": "[parameters('effect')]",
+      "details": {
+        "type": "Microsoft.ApiManagement/service/tenant",
+        "name": "access",
+        "existenceCondition": {
+          "allOf": [
+            {
+              "field": "Microsoft.ApiManagement/service/tenant/enabled",
+              "equals": "false"
```

Comment:
Shouldn't it be 'true' as you are writing existence condition na?

And also think if this condition needs to be added as well or not.
{
   "field": "Microsoft.ApiManagement/service/tenant/enabled",
   "exists": "true"
}

Reply:
Hi saurabh, no the value will be passed, because we pass the control if value if false. and in auditnot exist condition, we check the positive scenario.